index.html

```diff
@@ -4,14 +4,30 @@
 <meta name="viewport content="width=device-width, initial-scale="1.0">
 <meta http-equiv="X-UA-Compatible" contetn="ie=edge">
 <title> MainPage</title>
-<link rel="stylesheet" href="./main-page/style.css"> 
+<link rel="stylesheet" href="./main-page/style.css">
+<!-- <link rel="stylesheet" href="./style.css"> -->
     </head>
 
 <body>
         <div class="container">
-            <div clas="core-1">  <iframe src="https://www.figma.com/file/QTlwWm32d9QI2RcdMH0Ggy/project-3?node-id=0-1&t=T7AIaYQyP2iZAaoM-0" title="description"></iframe> </div>
-            <div clas="core-2"> <iframe src="https://www.figma.com/file/QTlwWm32d9QI2RcdMH0Ggy/project-3?node-id=0-1&t=T7AIaYQyP2iZAaoM-0" title="description"></iframe> </div>
-
```

Comment:
Figma embeds won't work unless you link to `"https://www.figma.com/embed?embed_host=astra&url=<figma-url-here>` and not the url directly